### ISSUE TITLE
total height not adding up to 1.0

### ISSUE THREAD

**Issue by kkj15dk** [jbkinney/logomaker#40]
Hi,

I might be using this package wrong, but I thought the default handling used probabilities directly as the heights in the constructed logoplots.
However, when using dataframes that are converted from torch.tensors using softmax (which I treat as probabilities), I get logoplots, where the total heights do not add up to 1, even though the input sums up to 1.0000

## minimal example:

import logomaker
import matplotlib.pyplot as plt
import torch
import pandas as pd

rand = torch.randn(20,64)
array = torch.nn.functional.softmax(rand, dim=1)
array_sums = torch.sum(array, dim=1)
print("sums", array_sums)

aminoacids = "ACDEFGHIKLMNPQRSTVWY"
amino_acids = list(aminoacids)

df = pd.DataFrame(array.T, columns=amino_acids, dtype=float)

fig, ax = plt.subplots(1,1,figsize=[4,2])
logo = logomaker.Logo(df, ax=ax)

## Output:
sums tensor([1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000,
        1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000,
        1.0000, 1.0000])

![image](https://github.com/user-attachments/assets/718e4631-0892-46c8-9f8f-b761baa1ca4b)

Expected output:
Something like figure 1C in (https://doi.org/10.1093/bioinformatics/btz921)

**Comment by kkj15dk** [comment]
Adding:
df = logomaker.transform_matrix(df, normalize_values=True)

after creating the dataframe solves this, but I'm not sure why it is needed.

**Comment by jbkinney** [comment]
Hi @kkj15dk, sorry for the delay in getting back to you. The reason the heights are different is because, even though the sums in each *column* sum to one (i.e., df.sum(axis=0) is uniform), the sums in each *row* do not sum to 1 (i.e. df.sum(axis=1) is not uniform. It is the sums across the rows in the dataframe that give the total height of each column in the logo.
